### DISCUSSION TITLE
Sell me the completed hardware

### DISCUSSION THREAD

**Discussion by Trud09** [sanni/cartreader#304] [Need help]
Is anyone interested in selling me the completed hardware?

I am a software guy, not hardware.

**Comment by Dakkaron** [comment]
It is a very simple build, especially if you don't need all systems. But @Androxilogin sometimes builds boards for others.

**Comment by Androxilogin** [comment]
I apologize for my late response- holidays and such. As @Dakkaron said, while I do have a few available, building them is half the fun (It's also much cheaper).

**Comment by Trud09** [comment]
@Androxilogin No apologies needed on the delay and thank you for your reply.

I likely won't have the bandwidth to try building one of these for a long time but I would like something working much sooner.

What price range is it if I want all the readers?

**Reply by Androxilogin** [reply]
**[Here's the page](https://bit.ly/cartreader)**. I send them out with the SNES, Genesis (or Mega Drive; depending on where you're from), GBA (which reads GB/GBC as expected) & N64 slots. NES adapter made separately. As many ask, it does include the 12F629 chip for SA-1 titles as well as the extra capacitor for better compatibility and the clock generator. You will, however, have to provide your own SD card.
